content/en/cosign/key_management/hardware-based-tokens.md

```diff
@@ -10,7 +10,7 @@ This support is enabled through the [PIV protocol](https://csrc.nist.gov/project
 and the [go-piv](https://github.com/go-piv/piv-go) library, which is not included in the standard release. Use `make cosign-pivkey-pkcs11key`, or `go build -tags=pivkey,pkcs11key ./cmd/cosign`, to build `cosign` with support for hardware tokens.
 
 ---
```

Comment:
It looks like the markdownlint bot is complaining about these borders, they're probably not needed if this is being changed from a note to a regular section.

Reply:
Hi @cmurphy! I am happy to fix this, but to give you some more context:

1. There are ten errors that showed up in the markdown linter. This is one of them. 
2. However, I had previously fixed ten other errors. You see them in places like [here](https://github.com/sigstore/docs/pull/323/files#diff-f9581b0e3c92d599da118c73f8b01ef23d383ce4125ec9b1bd99370d5ab19605R150). 
3. When I fixed those ten there were another ten found. 

Basically I am not sure how many errors will eventually show up in chunks of ten, and I asked @haydentherapper whether it was worth chasing all of these down when I am not actually bringing in new errors and the check didn't technically fail. I opened issue #325 to investigate this. 

Let me know what you think.

Reply:
I only pointed this one out because it looks like it was introduced by this change - the note header went from a bolded NOTE to an actual level-2 header, and there is now an error about the heading being surrounded by a blank line, which would not have existed previously. I certainly agree that it's not worth chasing all the preexisting errors down, but it would be better not to introduce new ones.

Reply:
@cmurphy Gotcha! That makes sense. It just got lost in the noise. I will fix.